src/components/Calculator.js

```diff
@@ -10,49 +10,39 @@ const content = [
   ['0', '.', '='],
 ];
 
-// eslint-disable-next-line react/prefer-stateless-function
-export class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { obj: {} };
-    this.handleClick = this.handleClick.bind(this);
-  }
+export const Calculator = () => {
+  const [obj, setObj] = useState({});
 
-  handleClick(e) {
-    const { obj } = this.state;
-    this.setState({ obj: calculate(obj, e.target.textContent) });
-  }
-
-  render() {
-    const { obj } = this.state;
-    const { next, total } = obj;
-    return (
-      <div className="column">
+  const handleClick = (e) => {
+    setObj(calculate(obj, e.target.textContent));
+  };
+  const { next, total } = obj;
+  return (
+    <div className="column">
+      <div className="row">
+        <input type="digit" placeholder="0" value={next || total} />
+      </div>
+      {content.map((row) => (
+        // eslint-disable-next-line react/jsx-key
         <div className="row">
-          <input type="digit" placeholder="0" value={next || total} />
-        </div>
-        {content.map((row) => (
-          // eslint-disable-next-line react/jsx-key
-          <div className="row">
-            {row.map((item, i) => {
-              if (i === row.length - 1) {
-                return <button type="button" onClick={this.handleClick} className="right">{item}</button>;
-              }
-              if (row.length === 3 && i === 0) {
-                return (
-                  <button type="button" onClick={this.handleClick} className="w-2">{item}</button>
-                );
-              }
+          {row.map((item, i) => {
+            if (i === row.length - 1) {
+              return <button type="button" onClick={(e) => handleClick(e)} className="right">{item}</button>;
+            }
+            if (row.length === 3 && i === 0) {
               return (
-              // eslint-disable-next-line react/jsx-key
-                <button type="button" onClick={this.handleClick}>{item}</button>
+                <button type="button" onClick={(e) => handleClick(e)} className="w-2">{item}</button>
               );
-            })}
-          </div>
-        ))}
-      </div>
-    );
-  }
-}
+            }
+            return (
+              // eslint-disable-next-line react/jsx-key
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
Thank you, i have resolved it